src/backend/common/util.hpp

```diff
@@ -17,5 +17,13 @@
 std::string getEnvVar(const std::string &key);
 
 // Dump the kernel sources only if the environment variable is defined
-static const char* saveJitKernelsEnvVarName = "AF_JIT_KERNEL_TRACE";
 void saveKernel(const std::string& funcName, const std::string& jit_ker, const std::string& ext);
+namespace {
+const char* saveJitKernelsEnvVarName = "AF_JIT_KERNEL_TRACE";
+
+std::string int_version_to_string(int version) {
```

Comment:
A comment on intended usage is useful. Are the values in the formulation (1000 , 10) always fixed ?